sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManagerImpl.kt

```diff
@@ -0,0 +1,250 @@
+package com.mapbox.maps.viewannotation
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewTreeObserver
+import android.widget.FrameLayout
+import androidx.annotation.LayoutRes
+import androidx.asynclayoutinflater.view.AsyncLayoutInflater
+import com.mapbox.maps.*
+import com.mapbox.maps.plugin.ViewPlugin
+import java.util.concurrent.ConcurrentHashMap
+
+internal class ViewAnnotationManagerImpl(
+  private val mapView: MapView,
+  private val mapboxMap: MapboxMap = mapView.getMapboxMap()
```

Comment:
Is it valid to set mapboxMap different from `mapView.getMapboxMap()` ?

Reply:
no, it's not valid, mapView <-> mapboxMap

Reply:
Then maybe pass just mapView and declare `mapboxMap` outside of the constructor?

Reply:
Yep